### PR TITLE
Operator 1.0.1 blog announce

### PR DESCRIPTION
**Pull request by rigazilla**
Operator 1.0.1 blog post. 
DO NOT MERGE operator is not yet published on operatorhub

### CODE REVIEW

File: blog/2019-10-31-infinispan-operator-1.0.1.adoc [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+---
+
+= Infinispan Operator 1.0.1 “Chupacabra Trainer”
+
+Dear Infinispan community,
+
+we know you are happy with the new shining 10.0.0 Infinispan release, but if you are among those who are missing a new operator version for safetly running you Infinispan Chupachabra in the clound, this post is for you!
+
+=== Versioning and channels
+This is our first blog post about 1.0.x operator series (yeah, sorry 1.0.0 we forgot about you) and as you can notice there's no Alpha, Beta or CR label at the end of the release tag.  This because OperatorHub and Openshift Catalog only allow numeric version like Maj.Min.Mic and instead of labels we now use channel to indicate the stability of a release. We have 2 live channels at the moment for the Infinispan operator: `stable` and `dev-preview`. Current `stable` is 0.3.2 which is for the 9.x Infinispan cluster and current `dev-preview` is 1.0.1 which works with 10.x clusters.
```
Comment (tristantarrant) [inline blog/2019-10-31-infinispan-operator-1.0.1.adoc]: s/This because/This is because/

Comment (tristantarrant) [reply blog/2019-10-31-infinispan-operator-1.0.1.adoc]: s/numeric/numerical/

Comment (tristantarrant) [reply blog/2019-10-31-infinispan-operator-1.0.1.adoc]: s/use channel/use the channel/

**Comment by galderz** [comment]
@rigazilla Have you addressed @tristantarrant's comments?

**Comment by rigazilla** [comment]
> @rigazilla Have you addressed @tristantarrant's comments?

yep @galderz, changes are there

**Comment by tristantarrant** [comment]
Merged, thanks

**Pull request merged**